EasyReflectometry/calculators/wrapper_base.py

```diff
@@ -198,3 +202,21 @@ def get_item_value(self, name: str, key: str) -> float:
         item = self.storage['item'][name]
         item = getattr(item, key)
         return getattr(item, 'value')
+
+    def set_resolution_function(self, resolution_function: Callable[[np.array], float]) -> None:
+        """Set the resolution function for the calculator.
+
+        :param resolution_function: The resolution function
+        """
+        self._resolution_function = resolution_function
+
+
+def constant_resolution_function(constant: float) -> Callable[[np.array], float]:
+    return linear_spline_resolution_function([sys.float_info.min, sys.float_info.max], [constant, constant])
+
+
+def linear_spline_resolution_function(q_data_points: np.array, resolution_points: np.array) -> Callable[[np.array], float]:
```

Comment:
please add a description of what this method is doing - it isn't immediately obvious

Reply:
done